### ISSUE TITLE
Free Memory Chunks needs granularity

### ISSUE THREAD

**Issue by stevegiron** [pivotal-cf/docs-pcf-healthwatch#207]
The free memory chunks information does not distinguish between PAS/PAS for Windows. 
Would be nice to have these separated out per PAS type.

**Comment by kbakas-piv** [comment]
@oppegard Not quite sure what we should do here, this is a good suggestion however this is a feature request to separate out PAS Windows information from PAS Linux cells in the graphs, right?

**Comment by stevegiron** [comment]
Yes seperate. We don't have ISO segs in all our foundations, so the combined free memory chunks for Linux/Windows cells as currently displayed doesn't indicate the detailed picture.

**Comment by oppegard** [comment]
@kbakas-piv Yes, this is a feature request to break down free chunks into separate measurements: one for PAS (linux) and another one for PAS Windows.
